File: C99/백준/Bronze/2292. 벌집/벌집.c
#include <stdio.h>
int main(void) {
    int n;
    scanf("%d", &n);
    int honey = 1, layer = 1;
    while (n > honey) {
        honey += 6 * layer;
        layer++;
    }
    printf("%d\n", layer);
    return 0;
}

File: Java/백준/Bronze/2292. 벌집/벌집.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int honey = 1;
        int layer = 1;
        while (n > honey) {
            honey += 6 * layer;
            layer++;
        }
        System.out.println(layer);
    }
}
